### ISSUE TITLE
.if (eax || edx) && ecx

### ISSUE THREAD

**Issue by nidud** [Baron-von-Riedesel/JWasm#4]
http://masm32.com/board/index.php?topic=5896.msg62670#msg62670
```
	.486
	.model flat
	.code

	.if (eax || edx) && ecx
		nop
	.endif

	.if !(eax || edx) && ecx
		nop
	.endif

	.while (eax || edx) && ecx
		nop
	.endw

	.while !(eax || edx) && ecx
		nop
	.endw

	.repeat
		nop
	.until (eax || edx) && ecx

	.repeat
		nop
	.until !(eax || edx) && ecx

ifdef __ASMC__
	.switch
          .case  (eax || edx) && ecx : nop : .endc
          .case !(eax || edx) && ecx : nop : .endc
        .endsw
endif
	END

```

**Comment by Baron-von-Riedesel** [comment]
Thanks! I guess this bug should really be fixed before releasing a final v2.12.
I always suspected that there must be an error, because the "AND"-code  is so much simpler than the "OR"-code.

**Comment by Baron-von-Riedesel** [comment]
fixed in v2.13pre9
